Makefile

```
@@ -0,0 +1,6 @@
+.PHONY: all
+all:
+
+.PHONY: test
+test:
+	py.test
```

This is a typo or a some Makefile auto-magic?

```suggestion
	pytest
```

---

Ah, that's pip magic.